### PR TITLE
support stream configs in fhir store

### PR DESCRIPTION
**Pull request by xingao267**

### CODE REVIEW

File: templates/tfengine/components/resources/healthcare_datasets/main.tf [hunk 1]
```diff
@@ -66,6 +66,23 @@ module "{{resourceName . "name"}}" {
       }
       {{end -}}
 
+      {{if has . "stream_configs" -}}
+      stream_configs = [
+        {{range $k, $v := .stream_configs -}}
+        {
+          bigquery_destination = {
+            dataset_uri = {{hcl $v.bigquery_destination.dataset_uri}}
```
Comment (umairidris) [inline templates/tfengine/components/resources/healthcare_datasets/main.tf:74]: you could probably just do "{{$v.biguqery_destination.dataset_uri}}"

**Review by umairidris — APPROVED** [review]

File: templates/tfengine/components/resources/healthcare_datasets/main.tf (hunk 1 above)
Comment (xingao267) [reply templates/tfengine/components/resources/healthcare_datasets/main.tf]: Done

**Pull request merged**